floor_generator/lib/value_object/dao.dart

```diff
@@ -14,7 +14,7 @@ class Dao {
   final List<UpdateMethod> updateMethods;
   final List<DeletionMethod> deletionMethods;
   final List<TransactionMethod> transactionMethods;
-  final List<Entity> streamEntities;
+  final List<Queryable> streamEntities;
```

Comment:
We only need entities here for now.
```suggestion
  final List<Entity> streamEntities;
```